pkg/cli/internal/cmd/setup/setup.go

```diff
@@ -129,15 +129,30 @@ func runE(conf *rest.Config, log logr.Logger, cmd *cobra.Command, flags *flags)
 			Name: constants.KubeCarrierDefaultNamespace,
 		},
 	}
+
+	kubeCarrier := &operatorv1alpha1.KubeCarrier{}
+	if flags.ConfigFile != "" {
+		f, err := os.Open(flags.ConfigFile)
+		if err != nil {
+			return err
+		}
+		yamlErr := yaml.NewYAMLOrJSONDecoder(f, 4*1024).Decode(kubeCarrier)
+		if err := f.Close(); err != nil {
+			return err
+		}
+		if yamlErr != nil {
+			return err
+		}
+	}
 	if err := spinner.AttachSpinnerTo(s, startTime, fmt.Sprintf("Create %q Namespace", ns.Name), createNamespace(ctx, c, ns)); err != nil {
 		return fmt.Errorf("creating KubeCarrier system namespace: %w", err)
 	}
 
-	if err := spinner.AttachSpinnerTo(s, startTime, "Deploy KubeCarrier Operator", reconcileOperator(ctx, log, c, ns)); err != nil {
+	if err := spinner.AttachSpinnerTo(s, startTime, "Deploy KubeCarrier Operator", reconcileOperator(ctx, log, c, ns, kubeCarrier.Spec.LogLevel)); err != nil {
```

Comment:
I don't really understand here, why do we set operator's logLevel from KubeCarrier CRD?
I think the operator logLevel should be passed as a flag in `kubecarrier setup` from user, and then we set up the log level of KubeCarrier CRD from that.
WDYT?

Reply:
My biggest use case for this feature is debugging certain operators, or deploy whole solution with extra logging.

I'm fine with either implementation, though I slightly prefer reusing the KubeCarrier CRD defined one.

Reply:
Ok, I am fine with it